start.py

```diff
@@ -6,6 +6,7 @@
 import subprocess
 import time
 import sys
+import requests
```

Comment:
This line should be after ```sys.path.insert(0, 'lib')```, because the requests library is provided in the lib folder.